### PR TITLE
Teamru invite user

### PR DESCRIPTION
**Pull request by sailokesh845**
completed features for invite user via email

### CODE REVIEW

File: src/components/Dashboard/TeamViewer/ManageTeam.jsx [hunk 1]
```diff
@@ -40,6 +41,13 @@ function ManageTeam(props) {
         }));
         setSubmit(false);
     };
+    const handleInviteUserChange = e => {
+        const { value } = e.target;
+        setUserEmail(value);
+    };
+    const onInviteUserSubmit = () => {
+        profile.inviteUser(team_id, userEmail);
+    };
```
Comment (zain08816) [inline src/components/Dashboard/TeamViewer/ManageTeam.jsx]: If there is an error, we should tell the user what the issue is and/or why it was not able to be completed.

**Comment by shivanmodha** [comment]
@sailokesh845 whats the status of this?

File: src/components/Dashboard/TeamViewer/ManageTeam.jsx (hunk 1 above)
Comment (sailokesh845) [reply src/components/Dashboard/TeamViewer/ManageTeam.jsx]: > @sailokesh845 whats the status of this?

Currently working on implementing it

**Comment by sailokesh845** [comment]
@zain08816 ready for review

**Review by zain08816 — APPROVED** [review]
Looks great!

**Pull request merged**